scripts/releng/update-release-info.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# release.md
+####################################
```

Comment:
What about other documents -- `patch-releases.md` and `schedule.yaml`? Are we permanently moving those two documents to this repo?

Reply:
Yep, there's an accompanying PR in k/sig-release here: https://github.com/kubernetes/sig-release/pull/1577